src/mprk.jl

```diff
@@ -416,13 +399,17 @@ end
 """
     MPRK22(α; [linsolve = ...])
 
-The second-order modified Patankar-Runge-Kutta algorithm for (conservative)
+The second-order modified Patankar-Runge-Kutta algorithm for 
 production-destruction systems. This one-step, two-stage method is
 second-order accurate, unconditionally positivity-preserving, and linearly
 implicit. The parameter `α` is described by Kopecz and Meister (2018) and
 studied by Izgin, Kopecz and Meister (2022) as well as
 Torlo, Öffner and Ranocha (2022).
 
+The scheme was introduced by Kopecz and Meister for conservative production-destruction systems. 
+For nonconservative production–destruction systems we use the straight forward extension
+analogous to `MPE`.
```

Comment:
```suggestion
analogous to [`MPE`](@ref).
```

This adds a link to the other docstring in the online version of the docs.